apps/academy/src/widget/apply/ui/ApplyCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { cn } from '@repo/util'
+import Image from 'next/image'
+import { ReactNode } from 'react'
+
+import EllipseSvg from '../../../../public/svg/ellipse.svg'
```

Comment:
public 폴더 접근은 /public/svg/ellipse.svg 이렇게도 할 수 있을 거 같은데 상대 경로로 쓰신 이유가 있을까요?

Reply:
해당 방법은 `import` 문을 사용하여 정적 파일을 가져올 때 상대경로로 사용하던 버릇이 있어서 사용하였습니다. 해당 부분 수정하였습니다.

```suggestion
import EllipseSvg from '/public/svg/ellipse.svg'
```